main/build.gradle.kts

```diff
@@ -132,20 +142,7 @@ dependencies {
     testImplementation("org.mockito:mockito-core:3.1.0")
     testImplementation("org.robolectric:robolectric:4.3.1")
 }
-// ~/.gradle/gradle.properties
-if (project.hasProperty("keystoreFile") &&
-        project.hasProperty("keystorePassword") &&
-        project.hasProperty("keystoreAliasPassword")) {
-    android.signingConfigs.getByName("release") {
-        storeFile = file(project.properties["keystoreFile"] as String)
-        storePassword = project.properties["keystorePassword"] as String
-        keyPassword = project.properties["keystoreAliasPassword"] as String
-        keyAlias = project.properties["keystoreAlias"] as String
-    }
-} else {
-    logger.warn("Release signing config not found. Using debug signing instead.")
-    android.buildTypes.getByName("release").signingConfig = android.signingConfigs.getByName("debug")
-}
+
```

Comment:
You are removing the fallback to debug keystore if the release cannot be found. This break at least the travis build. Otherwise the changes look good.

Reply:
A fall back to debug signing in a release build feels strange to me. Do you have a specific reason to use the release build on travis. Using assembleUiDebug instead of assembleUiRelease would also solve this issue. 

We could also introduce a staging build type which uses the debug signing but doesn't have the option to include debug features/code.

Reply:
release and debug also have different compile options etc. E.g. -O0 and -O2 for native code. So you actually need to sometimes build release to test that stuff. So basically it is more convient for me.

Reply:
Just one last try ;)
What about adding:
```
buildTypes {
    maybeCreate("staging").apply {
        initWith(getByName("release"))
        signingConfig = signingConfigs.getByName("debug")
    }
}
```
The `initWith` call copies all the settings from the release build.

Reply:
> Just one last try ;)
> What about adding:
> 
> ```
> buildTypes {
>     maybeCreate("staging").apply {
>         initWith(getByName("release"))
>         signingConfig = signingConfigs.getByName("debug")
>     }
> }
> ```
> 
> The `initWith` call copies all the settings from the release build.

Hi @schwabe do you think this will also work for the native compile options? If so I'll add this to this merge request. If not I'll add the debug fall back in the release sign config

Reply:
I have gone an easier way:

```
            if (project.hasProperty("icsopenvpnDebugSign")) {
                logger.error("This is acceptable")
                signingConfig = android.signingConfigs.getByName("debug")
            } else {
                signingConfig = signingConfigs.getByName("release")
                logger.error("This is unacceptable")
            }
```

I am that this works and also save me from compiling a stageging when I do a `./gradlew build`

I think that is clean enough.

Reply:
Without the debug stuff of course.